packages/storybook/src/stories/Fonts.tsx

```diff
@@ -1,23 +1,31 @@
+import { secondary } from '@littlespoon/theme/lib/fonts'
 import { Fragment } from 'react'
 
 interface Props {
   [style: string]: {
     family: string
+    [variant: string]: string
     weight: {
       [weight: string]: number
     }
   }
 }
 
-const headingFontSize = '3.25rem'
-const headingText = 'Display 01'
-const bodyFontSize = '1.25rem'
 const bodyText = 'The quick brown fox jumps over the lazy dog'
 
 function fontFormat(font: string) {
   return font.replace(', sans-serif', '').replace(/'/g, '')
 }
 
+function displayText(style, variant) {
```

Comment:
Can we move helper functions below what's getting exported to match the pattern we have on the FE?

Reply:
Given that DS is different from FE and unless we can enforce this pattern, I think what we have is fine. What do we gain/lose from this change?

Reply:
It's perfectly fine, this might be more a personal preference, so it's a suggestion but I think I can make an argument as to why we've done this on the FE and why it might be a good idea to keep it for other repos.

Here is what you get:
- Consistency across our FE projects. Even if it's a new one, I think it's better if it follows the same patterns we have on the others if possible.
- Not having to track down the exported file by scrolling or doing a search. 
The bigger the file gets, the more work to find it. By prioritizing public code to be at the beginning of the file, you get a consistent experience where the main code is always at the top.
- Not being forced into reading implementation details that you might not care about. My approach to code is to read the higher level first, and if I need to understand how a private fn works, then I'll jump into it and read it. You could ignore this, but it's pretty common to read top to bottom, so you'll end up reading the little pieces first and then the bigger picture.
- Easy distinction as to what is exported and what is a private helper. Obviously, the `export` keyword makes this explicit, but you get the main export first, then secondary exports and private functions last, so there is a structure to the file.

WDYT?

Reply:
Thanks for the explanation. I believe if we want to promote readability, we can always create a separate utils file. For now, I think the file is pretty small, we can always refactor later once it gets larger.

Reply:
Yeah, we can always refactor it later.